project/src/views/new-evaluation/technicalLevelPicker.js

```diff
@@ -0,0 +1,34 @@
+const TechnicalLevelPickerHeader = function (options = {}) {
+    const headers = options.map(h => `<th class="tbl-head tbl-head-skin">${h}</th>`);
```

Comment:
try to use variable names that are relevant to what the variable represents (even in map loops)

Reply:
I thought about 'headers' because that is what my constant will contain: a string of all headers of my table :D